sample-queries/sample-queries.json

```diff
@@ -1823,6 +1823,17 @@
         "postBody": "{\r\n  \"applicationName\": \"LinkedResource created from Microsoft Graph Explorer\"\r\n}",
         "tip": "This query requires the Tasks.ReadWrite permission. To find a value for the taskListId parameter, you can run: GET https://graph.microsoft.com/beta/me/todo/lists. to find a value for the taskId, you can run: GET https://graph.microsoft.com/beta/me/todo/lists/{taskListId}",
         "skipTest": false
-    }
+    },
+    {
+        "id": "",
+        "category": "Identity and Access",
+        "method": "GET",
+        "humanName": "Count the users in your tenant",
+        "requestUrl": "/beta/users/$count",
+        "headers": [ { "name": "ConsistencyLevel", "value": "eventual" } ],
```

Comment:
There shouldn't be a `headers` property for `GET` sample queries

Reply:
@irvinesunday my query requires a request header, [see example](https://developer.microsoft.com/en-us/graph/graph-explorer?request=users%2F%24count&method=GET&version=beta&GraphUrl=https://graph.microsoft.com&headers=W3sibmFtZSI6IkNvbnNpc3RlbmN5TGV2ZWwiLCJ2YWx1ZSI6ImV2ZW50dWFsIn1d).
How do I model this?

Reply:
@Licantrop0 that is fine, it will be added to the list of  headers. It can also be helpful the Content-type header to the sample

Reply:
@thewahome will GE recognize a header in a GET query? This is needed for Luca's sample.

Reply:
Yes, it will be placed in the request headers